doc/sphinx/proof-engine/tactics.rst

```diff
@@ -4243,10 +4244,10 @@ some incompatibilities.
       congruence.
       Qed.
 
-.. tacv:: congruence n
+.. tacv:: congruence @num
 
-  Tries to add at most `n` instances of hypotheses stating quantified equalities
-  to the problem in order to solve it. A bigger value of `n` does not make
+  Tries to add at most :ident:`num` instances of hypotheses stating quantified equalities
+  to the problem in order to solve it. A bigger value of :ident:`num` does not make
```

Comment:
You confused `:token:` with `:ident:` here. This is what makes the build fail.

Reply:
every. single. time. Sorry for the trouble.